### PR TITLE
WIP rfc8288 obsoletes rfc5988

### PR DESCRIPTION
**Pull request by ioggstream**
## This PR

updates reference to the latest Web Linking rfc 8288

### CODE REVIEW

File: chapters/references.adoc [hunk 1]
```diff
@@ -18,7 +18,7 @@ This section collects links to documents to which we refer, and base our guideli
 * *{RFC-3339}[RFC 3339]:* Date and Time on the Internet: Timestamps
 * *{RFC-4122}[RFC 4122]:* A Universally Unique IDentifier (UUID) URN Namespace
 * *{RFC-4627}[RFC 4627]:* The application/json Media Type for JavaScript Object Notation (JSON)
-* *{RFC-5988}[RFC 5988]:* Web Linking
+* *{RFC-8288}[RFC 8288]:* Web Linking
```
Comment (whiskeysierra) [inline chapters/references.adoc]: This should change usages of that reference as well. See https://github.com/zalando/restful-api-guidelines/search?q=RFC-5988&unscoped_q=RFC-5988

Comment (ioggstream) [reply chapters/references.adoc]: Sure. As I've no time I opened a draft-pr so that one day I'll remember :D

**Comment by ioggstream** [comment]
@whiskeysierra FYI I just received this https://github.com/sdatspun2/deprecation-header/blob/master/draft.md 

While these guidelines forbid `Link` headers, you may be interested in the `Deprecation` and `Sunset` header proposal.

**Comment by tkrop** [comment]
@ioggstream Thanks for notifying us. I will take over.

**Comment by ioggstream** [comment]
@tkrop Ok, sorry :)

**Comment by tkrop** [comment]
Replaced by #511.

**Comment by tkrop** [comment]
@ioggstream No prob. Thanks for the deprecation link, too. Do you know, whether this has or will be specified in an RFC?

**Comment by ioggstream** [comment]
@tkrop they will propose that to the IETF.  Comments are welcome.

if you are interested, I'm working on `Digest` :P https://ioggstream.github.io/draft-polli-resource-digests-http/draft-polli-resource-digests-http.html

**Pull request closed without merging**